README.md

```diff
@@ -1,7 +1,479 @@
 
 # income-tax-cis-frontend
 
-This is a placeholder README.md for a new repository
+This is where users can review and make changes to the CIS section of their income tax return.
+
+## Running the service locally
+
+you will need to have the following:
+- Installed [MondoDB](https://docs.mongodb.com/manual/installation/)
+- Installed/configured [service-manager](https://github.com/hmrc/service-manager)
+
+The service manager profile for this service is:
+
+    sm --start INCOME_TAX_CIS_FRONTEND
+Run the following command to start the remaining services locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+To run the service locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+    sm --stop INCOME_TAX_CIS_FRONTEND
+    ./run.sh **OR** sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes run
+
+This service runs on port: `localhost:9338`
+
+### Feature Switches
+
+| Feature | Environments Enabled In |
+| --- | --- |
+| Encryption | None|
+| Welsh Toggle | QA, Staging |
+| NRS | Local, QA, Staging, ET, Production |
+| taxYearError | Production |
+
+### CIS Sources (Contractor and Customer Data)
+CIS data can come from different sources: Contractor and Customer. Contractor data is CIS data that HMRC have for the user within the tax year, prior to any updates made by the user. The CIS data displayed in-year is Contractor data.
+
+Customer data is provided by the user. At the end of the tax year, users can view any existing CIS data and make changes (create, update and delete).
+
+<details>
+<summary>Click here to see an example of a user with Contractor and Customer data (JSON)</summary>
+
+```json
```

Comment:
Personally I think we should add a link to 'income-tax-submission-stub'. This will avoid duplication and we do not need to update this document when the user is updated.